File: janus/janus-extras/modules/scriptedagent/jythonengine/src/test/resources/org/janusproject/jythonengine/agent2.py
def activateAgent(agent) :
	agent.scriptedActivateExecuted = True

def liveAgent(agent) :
	if (agent.scriptedLiveExecuted) :
		agent.killMe()
		agent.scriptedKilledExecuted = True
	else :
		agent.scriptedLiveExecuted = True

def endAgent(agent) :
	agent.scriptedEndExecuted = True

File: janus/janus-extras/modules/scriptedagent/javascriptengine/src/test/resources/org/janusproject/javascriptengine/agent2.js
function activateAgent(agent)
{
	agent.scriptedActivateExecuted = true;
}

function liveAgent(agent)
{
	if (agent.scriptedLiveExecuted == true) {
		agent.killMe();
		agent.scriptedKilledExecuted = true;
	}
	else {
		agent.scriptedLiveExecuted = true;
	}
}

function endAgent(agent)
{
	agent.scriptedEndExecuted = true;
}
